libs/core/langchain_core/schema/embeddings.py

```diff
@@ -0,0 +1,22 @@
+from abc import ABC, abstractmethod
+from typing import List
+
+
+class EmbeddingsInterface(ABC):
```

Comment:
This matches the existing BaseClass:

https://github.com/langchain-ai/langchain/blob/b8f83d7f076098e10d001e67c782acc2e4c96a42/libs/core/langchain_core/embeddings.py#L20-L20

And I think we do want a default implementation of async methods that can be over-ridden by the implementer. Many developers aren't comfortable with async and will not know how to implement this, so at least their implementation will get pseudo support for async with the existing implementation in Embeddings